depends/packages/qt.mk

```diff
@@ -130,7 +133,7 @@ endif
 # for macOS on Apple Silicon (ARM) see https://bugreports.qt.io/browse/QTBUG-85279
 $(package)_config_opts_aarch64_darwin += -device-option QMAKE_APPLE_DEVICE_ARCHS=arm64
 
-$(package)_config_opts_linux = -qt-xcb
+$(package)_config_opts_linux = -xcb
```

Comment:
With getting rid of a `-qt-xcb` [crutch](https://bugreports.qt.io/browse/QTBUG-74874?focusedCommentId=458897&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-458897) it's possible to drop the related workaround as well: https://github.com/bitcoin/bitcoin/blob/fe03f7a37fd0ef05149161f6b95a25493e1fe38f/build-aux/m4/bitcoin_qt.m4#L152-L153

Tested on Fedora 35.

Reply:
Dropped for now, but I haven't tested this yet.